airflow/providers/airbyte/hooks/airbyte.py

```diff
@@ -71,7 +71,7 @@ def wait_for_job(
                 raise AirflowException(f"Timeout: Airbyte job {job_id} is not ready after {timeout}s")
             time.sleep(wait_seconds)
             try:
-                job = self.get_job(job_id=job_id)
+                job = self.get_job(int(job_id=job_id))
```

Comment:
I think you mean
```suggestion
                job = self.get_job(job_id=int(job_id))
```

Reply:
Yes, I was quick to mark the PR as draft because I noticed this :)